ckanext/s3filestore/views/resource.py

```diff
@@ -34,6 +34,10 @@
     url_defaults={u'package_type': u'dataset'}
 )
 
+PREVIEW_TYPES = [
+    'text/html',
+    'application/pdf',
```

Comment:
I have explicitly added `application/pdf` here, but maybe it would be better to always do do this if the `?preview=1` flag is set, regardless of content type, instead of maintaining a list of allowed types?

Reply:
I agree that it would be better to check the only the preview flag, please feel free to make the changes.

Reply:
:+1: done in ba65010